master.key

```diff
@@ -0,0 +1 @@
+93ca6e6484a46c2582754f54b5820c49
```

Comment:
🤯 🤯 🤯 🤯

Reply:
You have to generate a new master key and remove this

Reply:
😳 The `master.key` file is added to `.gitignore` as default. Please don't remove stuff from that file that you don't really understand. t's a google search or a question in Slack away.